addons/l10n_mn/data/account.tax.group.csv

```diff
@@ -0,0 +1,5 @@
+id,name
+tax_group_no_vat_0,No VAT
+tax_group_0,Zero VAT
+tax_group_10,VAT 10%
+tax_group_export_vat_0,Exporting VAT 0%
```

Comment:
Should not we just have one tax group with 0% instead of 3?

Reply:
Yes, We Can Do it If tax report no need separate value for this three 0% tax.